packages/amplify-cli/src/telemetry/index.ts

```diff
@@ -0,0 +1 @@
+export { getConfig } from './getConfig';
```

Comment:
It would be nice to export everything that is accessible from this folder in index to not to import files directly at other parts of the CLI.